docs/recipes/build/rollup.md

```diff
@@ -0,0 +1,135 @@
+# [Rollup](https://github.com/rollup/rollup) Bundling
+
+## Dependencies
```

Comment:
There needs to be a title and intro that explains what this doc is for / going to achieve.